lib/iex/lib/iex/introspection.ex

```diff
@@ -137,12 +137,6 @@ defmodule IEx.Introspection do
 
   defp open_abstract_code_reduce(entry, {file, module_pair, fa_pair}, fun, arity) do
     case entry do
-      {:attribute, _, :file, {ann_file, _}} ->
```

Comment:
this clause was causing issues when doing: `iex> open :erl_tar.create`. It switched file from `erl_tar.erl` to `filename.hrl` when reducing on chunks and that file is not present. I wasn't sure how else to make it work for `:erl_tar`

Reply:
Looks good to me! Let's make sure to test it too.

Reply:
looks like the existing test for `:httpc` handles this scenario (failing on some `.hrl`) so I think we're good :+1: